rg_shipit/shipit_service/ShipitHttpClient.php

```diff
@@ -0,0 +1,52 @@
+<?php
+  use GuzzleHttp\Client;
+
+  class ShipitHttpClient {  
+    public $endpoint = '';
+    public $headers = array();
+    public $client;
+    
+    public function __construct($endpoint, $headers) {
+      $this->endpoint = $endpoint;
+      $this->headers = $headers;
+      $this->client = new Client();
+    }
+
+    public function get() {
+      $response = $this->client->get( $this->endpoint
+        , ['headers' => $this->headers
+        ,'allow_redirects'=> ['strict'=>true]
```

Comment:
```suggestion
      $response = $this->client->get(
        $this->endpoint,
        ['headers' => $this->headers,
        'allow_redirects'=> ['strict'=>true]
```
Maybe align the function options to be more readable?

Reply:
done!